site/components/src/main/java/uk/nhs/hee/web/beans/PublicationLandingPage.java

```diff
@@ -34,19 +32,14 @@ public Calendar getPublicationDate() {
         return getSingleProperty("hee:publicationDate");
     }
 
-    @HippoEssentialsGenerated(internalName = "hee:publicationType")
-    public String getPublicationType() {
-        return getSingleProperty("hee:publicationType");
+    @HippoEssentialsGenerated(internalName = "hee:publicationTypeClassifiable")
+    public String[] getPublicationTypeClassifiable() {
+        return getMultipleProperty("hee:publicationTypeClassifiable");
```

Comment:
I seem to remember that this always needs to be supplied as a multi because it's using a taxonomy plugin. By default, taxonomy plugins always return lists, not single items. Even though I extended the plugin, I didn't want to adjust the return type, hence the "multi" approach.

Reply:
It might be possible in a future ticket to split these down into single and multis .. I don;t see why that could not happen, it just requires a bit of work to validate that approach

Reply:
Okay, that's fine